cluster/redis-pub-sub.go

```diff
@@ -32,36 +31,34 @@ func NewRedisPubSub(ctx context.Context, options *redis.Options) (*RedisPubSub,
 }
 
 // Logger returns the logger to use
-func (r *RedisPubSub) Logger() zLogger.GormLoggerInterface {
+func (r *RedisPubSub) Logger() *zerolog.Logger {
 	return r.logger
 }
 
 // Subscribe to a channel
 func (r *RedisPubSub) Subscribe(channel Channel, callback func(data string)) (func() error, error) {
-
-	ctx := context.Background()
 	channelName := r.prefix + string(channel)
 
 	if r.debug {
-		r.Logger().Info(ctx, fmt.Sprintf("NEW SUBSCRIPTION: %s -> %s", channel, channelName))
+		r.Logger().Info().Msgf("NEW SUBSCRIPTION: %s -> %s", channel, channelName)
```

Comment:
If `r.debug` is true, it good idea to use `Debug()` also. WDYT ?

Reply:
Done